source/sync/data-model/sync-schema-overview.txt

```diff
@@ -13,160 +13,58 @@ Sync Schema Overview
-       - array
-       - object
-
-       .. seealso::
-
-          For more information on supported data types, see the
-          documentation on :ref:`Schema Types <schema-types>`.
-
-   * - Optional
-     - Boolean
-     - Optional properties may contain a null value or be entirely
-       omitted from an object. By default, all properties are optional 
-       unless explicitly marked as required.
-
-   * - Default
-     - Boolean
-     - If a client application creates a new object that does not have a
-       value for a defined property, the object uses the default value
-       instead.
-
-       When you attempt to create an object that is missing a value
-       for a required field, it fails validation and does not persist
-       to the {+realm+}.
-
-   * - Indexed
-     - Boolean
-     - A property index significantly increases the speed of certain
-       read operations at the cost of additional overhead for write
-       operations. Indexes are particularly useful for equality
-       comparison, such as querying for an object based on the value of
-       a property. However, indexes consume additional storage.
-
-Summary
--------
-- App Services uses two different data models: a {+frontend-schema+} for 
-  mobile and an App Services Schema for Atlas. Changes to one data model match
-  the other data model. 
-- If you already have data in Atlas, App Services creates an App Services Schema 
-  by sampling that data. That App Services Schema can be translated into a Realm 
-  Object Model to be used in your App Services mobile application code. 
-- If you do not have data in Atlas or are developing with a mobile-first 
-  approach, you can turn on :guilabel:`Development Mode` to allow for data 
-  model changes from an App Services mobile client. When you finish developing your 
-  {+frontend-schema+}, you can turn off :guilabel:`Development Mode`, and 
-  your App Services Schema will be auto-updated with your updated data model 
-  configuration. Atlas will begin using this updated data model configuration 
-  for data validation on your cluster immediately.
+To use Alas Device Sync you must define your data model in two formats:
+
+- **App Services Schema**: A server-side schema which defines your
+  data in BSON. Device Sync uses the App Services Schema to convert your data
+  to MongoDB documents and enforce validation and synchronize data between client
+  devices and Atlas.
+
+- **Realm Object Schema**: Client-side schema of data defined using the Realm SDKs.
+  Each Realm SDK defines the Realm Object Schema in its own language-specific way.
+  The Realm SDKs use this schema to store data in Realm Database
+  and synchronize data with Device Sync.
+
+The App Services Schema and Realm Object Schema must be consistent with each other
+to sync data using Device Sync.
+
+You can also sync data between clients using different Realm SDKs,
+each of which implements the same Realm Object Schema. These Realm Object Schemas
```

Comment:
The "each of which..." construction feels weird in this case. Maybe "as long as they implement"...?